docs/proposals/network-policy.md

```diff
@@ -163,6 +191,16 @@ type NetworkPolicySpec struct {
 	// If this field is present and contains at least one rule, this policy allows any traffic
 	// which matches at least one of the ingress rules in this list.
 	Ingress []NetworkPolicyIngressRule `json:"ingress,omitempty"`
+
+	// List of egress rules to be applied to the selected pods.
+	// Traffic is allowed to exit a pod if namespace.networkPolicy.egress.isolation is undefined and cluster policy allows it,
+	// OR if the traffic destination is the pod's local node, 
+	// OR if the traffic matches at least one egress rule across all of the NetworkPolicy 
```

Comment:
Should we list that egress could permit traffic which is allowed based on ingress rules (reply packets) even though there is no allowed egress rule?